modules/core/src/types/types.ts

```diff
@@ -8,6 +8,7 @@ export interface ConstructorOf<T> {
 
 import {ShaderModule as _ShaderModule} from '@luma.gl/shadertools';
 
+// TODO remove and using type from luma.gl directly with generics
```

Comment:
@ibgreen do you agree? I feel that extending the type here makes things more confusing and means that the uniforms in the various deck.gl shader modules are untyped

Reply:
Yes - Perhaps this "alias" was added to compensate for lack of proper types in luma.gl? Now that we do have strong luma types, it makes sense to try to use those.